test/v1/validation/UltimateParentValidatorSpec.scala

```diff
@@ -125,90 +153,77 @@ class UltimateParentValidatorSpec extends BaseSpec {
         "Both UK Company and Partnership Details Supplied" in {
 
           val model = ultimateParentModelUkCompany.copy(sautr = Some(sautr))
-
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUkCompanyAndPartnership(model).errorMessage
         }
 
 
         "Both UK Company and NonUK Details Supplied" in {
 
           val model = ultimateParentModelUkCompany.copy(countryOfIncorporation = Some(nonUkCountryCode))
-
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUKCompanyAndNonUK(model).errorMessage
         }
 
         "Both UK Company and NonUK Details Supplied when uk is false" in {
 
           val model = ultimateParentModelUkCompany.copy(isUk = false, countryOfIncorporation = Some(nonUkCountryCode))
-
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUKCompanyAndNonUK(model).errorMessage
         }
 
         "Both UK Partnership and NonUk Details Supplied" in {
 
           val model = ultimateParentModelUkPartnership.copy(countryOfIncorporation = Some(nonUkCountryCode))
-
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUkPartnershipAndNonUKCompany(model).errorMessage
         }
 
         "Both UK Partnership and NonUk Details Supplied when uk is false" in {
 
           val model = ultimateParentModelUkPartnership.copy(isUk = false, countryOfIncorporation = Some(nonUkCountryCode))
-
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUkPartnershipAndNonUKCompany(model).errorMessage
         }
 
         "IsUK and CountryOfIncorporation is supplied" in {
 
-          val model = ultimateParentModelNonUkCompany.copy(
-            isUk = true
-          )
-
+          val model = ultimateParentModelNonUkCompany.copy(isUk = true)
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUkPartnershipAndNonUKCompany(model).errorMessage
         }
 
         "IsUk is false and SAUTR is supplied by default" in {
 
-          val model = ultimateParentModelUkCompany.copy(
-            isUk = false
-          )
-
+          val model = ultimateParentModelUkCompany.copy(isUk = false)
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUkPartnershipAndNonUKCompany(model).errorMessage
         }
 
         "NonUK and SAUTR is supplied" in {
 
-          val model = ultimateParentModelUkCompany.copy(
-            isUk = false,
-            ctutr = None,
-            sautr = Some(UTRModel("1123456789"))
-          )
+          val model = ultimateParentModelUkCompany.copy(isUk = false,
+            ctutr = None, sautr = Some(UTRModel("1123456789")))
 
           leftSideError(model.validate).errorMessage shouldBe WrongUltimateParentIsUkPartnershipAndNonUKCompany(model).errorMessage
         }
 
-        "NonUK and CTUTR is supplied" in {
+        "NonUK and CTUTR is supplied explicitly" in {
```

Comment:
Same as the first comment, I'm not sure what the difference in these tests are when we're doing the explicit and by default

Reply:
See above.